### PR TITLE
data: Badbatunde

### PR DESCRIPTION
**Pull request by Badbatunde**

### CODE REVIEW

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

File: data/Badbatunde.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Habeeb B. Oyesile",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Tech, Open-source, Finance and Books",
+  "avatar": "https://github.com/Badbatunde/png",
```
Comment (Pradumnasaraf) [inline data/Badbatunde.json:6]:
```suggestion
```

**Review by Pradumnasaraf — DISMISSED** [review]
Hey,

In the recent update, there is no need to add the `avatar:` key anymore. It will auto-pick your GitHub profile picture. So, please remove that. 

- [Updated docs]( https://linkfree.eddiehub.io/docs/quickstar)

If you need any help feel free to ask.

File: data/Badbatunde.json (hunk 1 above)
Comment (Badbatunde) [reply data/Badbatunde.json]: The change has been made as requested.

The update docs link you attached displayed a '404: This page cannot be found' error by the way.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by Badbatunde** [comment]
Kindly visit my changes and approve, thanks.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**